### ISSUE TITLE
Crash when frame are not loaded by a player with funky locomotion beta7 

### ISSUE THREAD

**Issue by zed-hex** [rwtema/FunkyLocomotion#86]
Crash when frame are not loaded by a player with funky locomotion beta7 

If a frame machine is moving when a player is not near it the server or client will crash dependent on if this is on a server or single player world. Here is my quick test frame machine I made to test versus different conditions.  http://prntscr.com/9g07xu 
I know in the previous build (6a) a bug similar to this was fixed and I have tested with the same frame machine in build (6a) and it is fine except any players that are near but not in viewing distance of the frame machine will get extremely large lag spikes every time the frame machine moves.
Here is the full crash log: http://pastebin.com/nggpJnK1

**Comment by SamDiego** [comment]
Yeah same thing happening here, if I leave the frame setup when it moves the client OR server crashes.

http://pastebin.com/y8bwMGsW
http://pastebin.com/SEEvjG5x

**Comment by mechoriet** [comment]
same here

**Comment by popcorn9499** [comment]
I have the same issue almost exactly the same log too. I was wondering if it was just in a different dimension but that doesn't seem to be the case

**Comment by Pontiac76** [comment]
Kind of a necro thread, but, I'm getting the same thing.  I'm in another dimension, using wireless redstone from WRC (I think thats it?) to control the frame via ComputerCraft.  At random intervals, sometimes as soon as the computer sends the signal, other times, after 5 or 6 runs, the server dies hard.

I'm in 1.7.10 playing a modified version of Infinity on a LAN.

**Comment by TheInfamousRaoul** [comment]
I am getting the same thing on a server. I've been remotely controlling the frame (from the same dimension) with computercraft/Chickenbones' wireless redstone. It is fine as long as a player is loading the chunks but as soon as the player stops loading the chunks and it's just a chunkloader, the server crashes.
Playing FTB Infinity Evolved Expert (v 2.5.0, Funky Locomotion 1.7.10-beta-7)

Crashlog here: [http://pastebin.com/SNQnGdDW](http://pastebin.com/SNQnGdDW)

**Comment by popcorn9499** [comment]
you guys realise how old this thread is?

**Comment by cjburkey01** [comment]
If you build the mod yourself, I think the issue is fixed.

**Comment by nmills3** [comment]
I know i'm a bit late but I think that i have a fix for this issue. Downgrade to beta 6a, you will lose frame boosters but it's better than crashing. Hope this helps someone.
